src/API.jl

```diff
@@ -1560,14 +1563,13 @@ function instantiate(ctx::Context; manifest::Union{Bool, Nothing}=nothing,
     # Handling packages tracking repos
     for pkg in pkgs
         pkg.repo.source !== nothing || continue
+        repo_source = pkg.repo.source::String
```

Comment:
Move this up and use in the `!== nothing` check?

Reply:
Done 👍